router/core/attribute_initialization.go

```diff
@@ -0,0 +1,13 @@
+package core
+
+import (
+	"github.com/wundergraph/cosmo/router/pkg/config"
+	"go.uber.org/zap"
+)
+
+func initializeAttributes(logger *zap.Logger, attrs []config.CustomAttribute, enableAttributeMapper bool) (mapper *attributeMapper, expressions *attributeExpressions, err error) {
```

Comment:
I haven't seen it reused multiple places. I think we can remove it and copy it to the place. I don't see a benefit in creating a function for it.